tests/test_imgurfetcher.py

```diff
@@ -21,11 +25,19 @@ class test_imgurfetcher(unittest.TestCase):
     gallery = None
     albums = None
     mock_client = None
+    settings_path = None
+    bad_image_height = None
+    bad_image_width = None
+    bad_image_title = None
+    bad_image_description = None
+    good_image = None
+    fake_response = None
 
     def setUp(self):
 
-        settings_path = os.path.join(os.getcwd(), "tests",  "settings.json")
-        self.fetcher = imgurfetcher.imgurfetcher(settings_path)
+        self.settings_path = os.path.join(os.getcwd(), "tests",
```

Comment:
I would use something like:

```python
from os.path import abspath, dirname, join
join(dirname(abspath(__file__)), 'tests')
```
This way you guarantee you can get to the tests folder, regardless on where you are running the tests from.